content/en/docs/setup/install/multicluster/simplified/index.md

```diff
@@ -0,0 +1,169 @@
+---
+title: Simplified Multicluster Install [Experimental]
+description: Configure an Istio mesh spanning multiple Kubernetes clusters.
+weight: 1
+keywords: [kubernetes,multicluster]
+---
+
+{{< boilerplate experimental-feature-warning >}}
+
+This guide describes how to configure an Istio mesh that includes multiple Kubernetes clusters using a simplified experimental approach.
+We hope to continue developing this functionality in coming releases, so we'd love your feedback on the overall flow.
+
+We focus here on the details of getting a multicluster mesh wired up, refer to [multicluster deployment model](/docs/setup/deployment-models/#multiple-clusters) for
+additional background information. We'll show how to connect two clusters that are on the same network together, along
+with a third cluster that's on a different network.
+
+Using the approach shown in this guide results in an instance of the Istio control plane being deployed in every cluster
+within the mesh. Although this is a common configuration, other more complex topologies are possible, but have to be done
+using a more manual process, not described herein.
+
+## Before you begin
+
+The procedures we describe here are primarily intended to be used with relatively pristine clusters,
+where Istio hasn't already been deployed. We hope to expand support in the future to existing clusters.
+
+For the sake of explanation, this guide assumes you have created three Kubernetes clusters:
+
+- A cluster named `cluster-east-1` on the network named `network-east`.
+- A cluster named `cluster-east-2` on the network named `network-east`.
+- A cluster named `cluster-west-1` on the network named `network-west`.
+
+These clusters shouldn't have Istio on them yet. The first two clusters are on the same network and have
+direct connectivity, while the third cluster is on a different network.
+Take a look at the [platform setup instructions](/docs/setup/platform-setup)
+for any special instructions for your particular environment.
+
+## Initial preparations
+
+You need to do a few one-time steps in order to be able to setup a multicluster mesh:
+
+1. Ensure that all of your clusters are included in your [Kubernetes configuration file](https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/#define-clusters-users-and-contexts)
+and create contexts for each cluster. Once you're done, your configuration file should include something similar to:
+
+    {{< text syntax="yaml" downloadas="kubeconfig.yaml" >}}
+    kind: Config
+    apiVersion: v1
+    clusters:
+    - cluster:
+      name: cluster-east-1
+    - cluster:
+      name: cluster-east-2
+    - cluster:
+      name: cluster-west-1
+    contexts:
```

Comment:
This doesn't appear to show correctly. I see `contexts:` on the end of the prior line.

Reply:
Could you send me a screenshot? It seems to render OK for me.

What browser are you using?

Reply:
![nAzv23QzcEF](https://user-images.githubusercontent.com/116378/68816877-b3090400-0634-11ea-9634-42847790d891.png)

Reply:
The rendered output is better. The `cluster:` and `context:` array elements need to be unindented left two spaces.

![Screenshot from 2019-11-13 16-44-30](https://user-images.githubusercontent.com/116378/68816927-e481cf80-0634-11ea-893f-0c686f2d768f.png)

Reply:
I removed the indent.

You can't look at GitHub's rendering of the markdown to gauge whether it looks good or not. The only valuable test is to look at the netlify preview.

Reply:
I was probably using Chrome at the time, but tried again with Safari and Chrome and now it seems to display correctly.